roles/bindle-profiles/tasks/main.yml

```diff
@@ -16,26 +16,31 @@
   git: repo=https://github.com/CloudBindle/Bindle
        dest=/home/{{ user_name }}/architecture2/Bindle
        version={{ bindle_git_branch }}
+       force=no
```

Comment:
After watching the process take foreeeeeeeeeeeever on one of Adam's launchers, I added some code to do `git status` on the repos before ansible checks them out. This should speed the whole process up.